healpy/newvisufunc.py

```diff
@@ -46,13 +53,23 @@ def mollview(
     flip="astro",
     format="%g",
     cbar=True,
-    cmap=None,
+    cmap='jet',
     norm=None,
     graticule=False,
     graticule_labels=False,
+    return_only_data=False,
+    projection_type = "mollweide",
+    cb_orientation = "horizontal",
+    xlabel=None,
+    ylabel=None,
+    longitude_grid_spacing = 60,
+    latitude_grid_spacing = 30,
+    override_plot_properties=None,
+    title=None,
+    lcolor='black',
```

Comment:
can you please run `black` on the code so the formatting is consistent?

Reply:
done!